lib/ssl/src/ssl_connection.erl

```diff
@@ -668,9 +668,11 @@ user_hello({call, From}, cancel, #state{negotiated_version = Version} = State, _
                      Version, ?FUNCTION_NAME, State);
 user_hello({call, From}, {handshake_continue, NewOptions, Timeout}, #state{hello = Hello,
                                                                            role = Role,
-                                                                           start_or_recv_from = RecvFrom,
+                                                                           start_or_recv_from = _RecvFrom,
```

Comment:
You can remove this line as it no longer needed!

Reply:
Removed